src/Files.Uwp/UserControls/SidebarControl.xaml

```diff
@@ -65,17 +64,18 @@
                     SelectsOnInvoked="{x:Bind SelectsOnInvoked}"
                     Tag="{x:Bind Path}"
                     ToolTipService.ToolTip="{x:Bind HoverDisplayText}">
-                    <muxc:NavigationViewItem.Icon>
-                        <muxc:ImageIcon Source="{x:Bind Icon, Mode=OneWay}" />
-                    </muxc:NavigationViewItem.Icon>
-                    <muxc:NavigationViewItem.InfoBadge>
-                        <muxc:InfoBadge Style="{ThemeResource CautionIconInfoBadgeStyle}" Visibility="{x:Bind IsInvalid, Mode=OneWay}" />
-                    </muxc:NavigationViewItem.InfoBadge>
-                </muxc:NavigationViewItem>
+                    <NavigationViewItem.Icon>
+                        <ImageIcon Source="{x:Bind Icon, Mode=OneWay}" />
+                    </NavigationViewItem.Icon>
+                    <!--  WINUI3  -->
+                    <!--<NavigationViewItem.InfoBadge>
```

Comment:
Same question regarding info badge, we should add these to the WinUI progress issue.